pkg/runctl/controller.go

```diff
@@ -668,8 +704,11 @@ func (c *Controller) getWaitTimeout(pipelineRunsConfig *cfg.PipelineRunsConfigSt
 
 // TODO find better name
 func (c *Controller) handleResultError(ctx context.Context, pipelineRun k8s.PipelineRun, result api.Result, message string, err error) error {
+	logger := klog.FromContext(ctx)
+
 	pipelineRun.UpdateMessage(err.Error())
-	pipelineRun.StoreErrorAsMessage(err, message)
+	logger.V(3).Info("Follows error message")
```

Comment:
Not clear what "Follows error message" means.
May be give some context in StoreErrorAsMessage function instead.

Reply:
This is not necessary (I will remove it). I introduced before updating PipelineRun interface functions (major change is to add `context.Context` as the first function argument to some (non-`get...()` funcs) interface funcs.)